README.md

```diff
@@ -39,8 +39,11 @@ npm run action src/www/storybook
 Additional requirements for Android:
 
 - [Android Studio 2020.3.1+](https://developer.android.com/studio)
+  - Optional for building, but useful for development
 - [Latest Android Sdk Commandline Tools](https://developer.android.com/studio/command-line)
-- Android SDK 30 (with build-tools) via commandline `sdkmanager "platforms;android-30" "build-tools;30.0.3"`
+  - Place it at `~/Android/Sdk/cmdline-tools/latest`
+- Android SDK 30 (with build-tools) via commandline `~/Android/Sdk/cmdline-tools/latest/bin/sdkmanager "platforms;android-30" "build-tools;30.0.3"`
+  - Set up the environment: `export ANDROID_SDK_ROOT=~/Android/Sdk`
```

Comment:
```suggestion
  - Set up the environment: `export ANDROID_HOME=~/Android/Sdk`
```

Reply:
Well, Cordova seems to need ANDROID_SDK_ROOT, so I reverted that and added a note.

Reply:
`ANDROID_HOME` is deprecated: https://github.com/apache/cordova-android/issues/949

Reply:
`ANDROID_SDK_ROOT` is deprecated: https://developer.android.com/studio/command-line/variables :-)

Reply:
Cordova is just like, wrong then: https://github.com/Jigsaw-Code/outline-client/runs/6196839202?check_suite_focus=true#step:7:184